hw2/templates/result3.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<h1>привет {{ name }} !</h1>
+<form action="{{ url_for('clear') }}" method="post">
```

Comment:
Тут нужно будет убрать `action`
```suggestion
<form method="post">
```